account/models.py

```diff
@@ -0,0 +1,60 @@
+from django.db import models
+from product.models import *
+
+
+class User(models.Model):
+	first_name	= models.CharField(max_length = 100)
+	last_name	= models.CharField(max_length = 100)
+	email		= models.CharField(max_length = 200)
+	password	= models.CharField(max_length = 100)
+	prefix		= models.ForeignKey('Prefix',on_delete = models.SET_NULL, null=True)
+	created_at	= models.DateTimeField(auto_now_add = True)
+	updated_at	= models.DateTimeField(auto_now = True)
+	country		= models.CharField(max_length = 50)
+	address		= models.ManyToManyField('Address', through = 'UserAddress')
+	wishlist    = models.ManyToManyField(Product,through = 'UserWishlist')
+
+	class Meta:
+		db_table = 'users'
+
+class Address(models.Model):
+	name		= models.CharField(max_length = 50)
+	st_addr		= models.CharField(max_length = 300)
```

Comment:
```suggestion
	st_address		= models.CharField(max_length = 300)
```
st는 어떤 줄임일까요? 줄임말은 지양해주세요!